quests/argeas/soup_and_slimes.xml

```diff
@@ -8,14 +8,12 @@
   </its:rules>
   <var id="11">
     <effect map="009-2.tmx" npc="117" value="0" effect="45"/>
-    <effect map="009-2.tmx" npc="117" value="1,2,3,4,5" effect="46"/>
-    <effect map="008-1.tmx" npc="120" value="0,1,2,3" effect="45"/>
-    <effect map="008-1.tmx" npc="120" value="4,5" effect="46"/>
+    <effect map="009-2.tmx" npc="117" value="1,2,3" effect="46"/>
     <quest name="Soup" group="Argeas" incomplete="1">
       <name>Soup</name>
       <text>Bernard needs a Roasted Maggot</text>
     </quest>
-    <quest name="Soup" group="Argeas" incomplete="2">
+    <quest name="Soup" group="Argeas" complete="2">
```

Comment:
The 2 complete states are intended. You first complete the first part of the quest, then he shows an exclamation point and gives you the second part and then it is 100% complete.